cypress/e2e/course_board/openDrawingElementOnBoard.feature

```diff
@@ -1,25 +1,71 @@
 @pr
 @release
+@stable_test
+@api-migrated
 Feature: Course Board - Opening a drawing element on a course page
 
     As a teacher and student I want to open already created drawing element on a course page
 
-    @stable_test
-    Scenario Outline:  Teacher is able to open a drawing element to the course board
+    Scenario Outline: Users (admin, teacher, student), create a course with board and open TLDraw.
```

Comment:
there is a comma added after closing the bracket.

This like should be like -> Users (admin, teacher, student) create a course with board and open TLDraw, including pre-conditions

Reply:
Done